concent_api/core/tests/utils.py

```diff
@@ -625,3 +649,24 @@ def _add_signature_to_message(self, golem_message, priv_key):  # pylint: disable
         golem_message = golem_message.serialize(sign_func = signature)
         golem_message = message.Message.deserialize(golem_message, decrypt_func = None, check_time = False)
         return golem_message.sig
+
+    def _create_client_auth_message(self, client_priv_key, client_public_key):  # pylint: disable=no-self-use
+        client_auth = message.concents.ClientAuthorization()
+        client_auth.client_public_key = client_public_key
+        return dump(client_auth, client_priv_key, settings.CONCENT_PUBLIC_KEY)
+
+    def _create_provider_auth_message(self):
+        return self._create_client_auth_message(self.PROVIDER_PRIVATE_KEY, self.PROVIDER_PUBLIC_KEY)
+
+    def _create_diff_provider_auth_message(self):
+        return self._create_client_auth_message(self.DIFFERENT_PROVIDER_PRIVATE_KEY, self.DIFFERENT_PROVIDER_PUBLIC_KEY)
+
+    def _create_requestor_auth_message(self):
+        return self._create_client_auth_message(self.REQUESTOR_PRIVATE_KEY, self.REQUESTOR_PUBLIC_KEY)
+
+    def _create_diff_requestor_auth_message(self):
+        return self._create_client_auth_message(self.DIFFERENT_REQUESTOR_PRIVATE_KEY, self.DIFFERENT_REQUESTOR_PUBLIC_KEY)
+
+    def _create_test_ping_message(self):
+        self.ping_message = message.Ping()
+        return self.ping_message
```

Comment:
"C:670, 0: Final newline missing (missing-final-newline)"
There is a bunch of pylint errors that need to be resolved.

Reply:
Please check my comment above.